packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -845,7 +845,7 @@ class DropdownButton<T> extends StatefulWidget {
        assert(isDense != null),
        assert(isExpanded != null),
        assert(autofocus != null),
-       assert(itemHeight == null || itemHeight >=  kMinInteractiveDimension),
+       assert(itemHeight == null || kIsWeb || itemHeight >=  kMinInteractiveDimension),
```

Comment:
`kIsWeb` could be true on a mobile device..
This also does not seem to account for Desktop platforms. I assume the min height would not be desired there either.

Reply:
Hey @Piinks, actually I marked this as a draft as a result of the discussion on the issue : https://github.com/flutter/flutter/issues/69398#issuecomment-722698221
I was waiting for a proper resolution on the issue as to what should be the appropriate way to handle this.

Reply:
I see! Thanks for the update. :)
There does not appear to be any discussion, at least not for a few months now. Should we close this for now until there is something actionable?

Reply:
Yes, it's okay to close this as of now. If there's any update on the issue I'll pick it up in a new PR :).

Reply:
In my opinion, it should be removed altogether. Creates issues when writing truly responsive UIs. For example, I have an option in my app to reduce the scale factor of UI which makes sense for large life-size displays. If i do so, this dropdown assert cries. I think this should be handled the same way as popupmenu does - by not having any such constraints. Please reopen this issue.